src/batch/src/executor/insert.rs

```diff
@@ -353,6 +353,7 @@ mod tests {
                     table_id: Default::default(),
                     retention_seconds: None,
                     read_version_from_backup: false,
+                    prefetch_options: PrefetchOptions::new_for_exhaust_iter(false),
```

Comment:
Still not ideal... I sugget not leaking exhaust_iter by default. It is better to impl Default for `PrefetchOptions` and replace `PrefetchOptions::new_for_exhaust_iter(false)` with `Default::default()`. With this, we have a chance to change the default behavior without changing upper-level codes.

Reply:
> Still not ideal... I sugget not leaking exhaust_iter by default. It is better to impl Default for `PrefetchOptions` and replace `PrefetchOptions::new_for_exhaust_iter(false)` with `Default::default()`. With this, we have a chance to change the default behavior without changing upper-level codes.

fixed